pkg/signature/kms/cliplugin/common/interface_test.go

```diff
@@ -114,9 +145,10 @@ func TestPluginRespJSON(t *testing.T) {
 	// we fill them here for simpler testing.
 	testPluginResp := &PluginResp{
 		ErrorMessage:     "any error message",
-		DefaultAlgorithm: &DefaultAlgorithmResp{DefaultAlgorithm: "any-algo"},
-		CreateKey:        &CreateKeyResp{PublicKeyPEM: []byte("mypem")},
-		SignMessage:      &SignMessageResp{Signature: []byte("any-signature")},
+		DefaultAlgorithm: &DefaultAlgorithmResp{DefaultAlgorithm: testAlgorithm},
```

Comment:
I feel like this test should test closer to what's expected rather than one big chunk? Can we split it in to parameterized testcases within this test?

Reply:
changed.